### ISSUE TITLE
Add "from" and "to" on transactions

### ISSUE THREAD

**Issue by Powerade** [Electra-project/electra-js#17]

**Comment by ivangabriele** [comment]
It should normally be available since it returns `Promise<WalletTransaction[]>` with:

```
export interface WalletTransaction {
  amount: number;
  confimationsCount: number;
  date: number;
  from?: string[];
  hash: string;
  to: string[];
  type: WalletTransactionType;
}

export enum WalletTransactionType {
  GENERATED = 'GENERATED',
  RECEIVED = 'RECEIVED',
  SENT = 'SENT',
}
```

**Comment by ivangabriele** [comment]
Fixed in v0.14.4.
